docs/configuration/index.md

```diff
@@ -310,31 +311,36 @@ For native query, the following MDC fields are populated with `setMDC`:
 |`resultOrdering`|The ordering of results|
 |`descending`|If the query is a descending query|
 
-#### Filtered Request Logging
-Filtered Request Logger filters requests based on a configurable query/time threshold (for native query) and sqlQuery/time threshold (for SQL query).
-For native query, only request logs where query/time is above the threshold are emitted. For SQL query, only request logs where sqlQuery/time is above the threshold are emitted.
+#### Filtered request logging
+
+The `filtered` request logger filters requests based on the query type or how long a query takes to complete.
+For native queries, requests are only logged when the `query/time` metric is above the user-provided threshold.
+For SQL queries, requests are only logged when the `sqlQuery/time` metric is above the user-provided threshold.
```

Comment:
```suggestion
For SQL queries, it only logs requests when the `sqlQuery/time` metric exceeds the user-provided threshold.
```
for line 317 too,  is this a specific threshold we can mention. For example when sqlQuery/time exceeds the threshold set in <config>

Reply:
```suggestion
For SQL queries, it only logs requests when the `sqlQuery/time` metric exceeds threshold provided in `sqlQueryTimeThresholdMs`.
```